text-icu.cabal

```diff
@@ -82,12 +82,7 @@ library
       Data.Text.ICU.Spoof.Pure
       Data.Text.ICU.Text
   c-sources: cbits/text_icu.c
-  include-dirs: include
-  extra-libraries: icuuc
-  if os(mingw32)
-    extra-libraries: icuin icudt
-  else
-    extra-libraries: icui18n icudata
```

Comment:
We don’t actually need to link these here because we don’t actually link here at all. `ghc` builds object files for library packages, and not e.g. `.a` files, wherever possible, and then links them together into the final executable.